swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from uuid import uuid4
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, fabric = None, condition = 0):
```

Comment:
Spaces do not surround = when you're using default params. This same comment applies to all other places in your project where you have spaces surrounding = for default params.

```suggestion
    def __init__(self, id=None, fabric=None, condition=0):
```